Jenkinsfile

```diff
@@ -243,33 +243,6 @@ pipeline {
           }
         }
 
-        stage('S3') {
-          agent {
-            docker { image 'px4io/px4-dev-base-focal:2021-08-18' }
-          }
-          steps {
-            sh('export')
-            unstash 'metadata_airframes'
-            unstash 'metadata_parameters'
-            sh('ls')
-            withAWS(credentials: 'px4_aws_s3_key', region: 'us-east-1') {
-              s3Upload(acl: 'PublicRead', bucket: 'px4-travis', file: 'airframes.xml', path: 'Firmware/master/')
-              s3Upload(acl: 'PublicRead', bucket: 'px4-travis', file: 'parameters.xml', path: 'Firmware/master/')
-              s3Upload(acl: 'PublicRead', bucket: 'px4-travis', file: 'parameters.json.xz', path: 'Firmware/master/')
```

Comment:
Where were these data being used at anyways btw? I can't think of any case @bkueng

Reply:
QGC does not use them apparently.
Flight review uses the ones from `metadata.yml`: https://github.com/PX4/flight_review/blob/main/app/config_default.ini#L13.
(We can rename all to `main`).

So looks like we can just remove these.